resources/api-description-overrides.json

```diff
@@ -1,6 +1,582 @@
 [
   {
-    "identifier": "api-simulator",
-    "description": "API Simulator OVERRRIDE Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. "
+    "description": "Search for postal and BFPO (British Forces Post Office) address details within the UK and retrieve them in a common format",
+    "identifier": "address-lookup"
+  },
+  {
+    "description": "An API allowing MTD-enabled Agents to request authorisation to a service for a client, instead of filling the 64-8 paper form.",
+    "identifier": "agent-authorisation-api"
+  },
+  {
+    "description": "The Agent Authorisation Test Support API provides test-only ability to accept or reject an authorisation request created with Agent Authorisation API POST /agents/:arn/invitations.",
+    "identifier": "agent-authorisation-test-support-api"
+  },
+  {
+    "description": "A 'hello world' example of an API on the HMRC API Developer Hub.",
+    "identifier": "api-example-microservice"
+  },
+  {
+    "description": "Allows for retrieval of notifications",
+    "identifier": "api-notification-pull"
+  },
+  {
+    "description": "The Create Test User API is responsible for creating test users on the external test environment.",
```

Comment:
This should be "sandbox" not "external test". This is public facing where it's referred to as "sandbox".